SingularityServiceIntegrationTests/src/test/java/com/hubspot/singularity/SingularityStateIT.java

```diff
@@ -22,4 +22,10 @@ public void testStateEndpoint(SingularityClient singularityClient) {
 
     assertEquals(3, state.getActiveAgents());
   }
+
+  @Test
+  public void testScheduledTasksInfoEndpoint(SingularityClient singularityClient) {
+    final SingularityScheduledTasksInfo scheduledTasksInfo = singularityClient.getScheduledTasksInfo();
+    // TODO: assertion
+  }
```

Comment:
We don't actually have any real unit tests set up for the SingularityClient at the moment (yes... shame on us...). If you are trying to fit unit tests in for this PR, better bet would be to write one for the `stateManager.getScheduledTasksInfo();` method. The SingularityClient methods you'll likely end up testing in our staging environment instead

Reply:
Sounds good - I'll test the new client method in staging then and won't add the unit test here as `stateManager.getScheduledTasksInfo()` is not the functionality being added in this PR.